addons/airway/stringtable.xml

```diff
@@ -898,5 +898,53 @@
             <Italian>Colora gli Activitylogs di AED-X o il rosso del pulsossimetro, se un valore è critico</Italian>
             <Japanese>値が重要な場合は、AED-Xまたはパルスオキシメーターのアクティビティログを赤に着色します</Japanese>
         </Key>
+        <Key ID="STR_kat_airway_RecoveryPosition">
+            <English>Established in recovery position</English>
+            <Polish>Ułożony w pozycji bezpiecznej</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_RecoveryPosition_displayName">
+            <English>Establish recovery position</English>
+            <Polish>Ułóż w pozycji bezpiecznej</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_RecoveryPosition_displayNameProgress">
+            <English>Establishing recovery position</English>
+            <Polish>Układanie w pozycji</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_Recovery_Info">
+            <English>Recovery position established successfully</English>
+            <Polish>Pomyślnie ułożono w pozycji bezpiecznej</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_Recovery_Cancel">
+            <English>Recovery position cancelled</English>
+            <Polish>Przerwano pozycję bezpieczną</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_RecoveryPosition_Log">
+            <English>%1 established recovery position</English>
+            <Polish>%1 ułożył w pozycji bezpiecznej</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_TIME_RECOVERY">
+            <English>Time for recovery position</English>
+            <Polish>Czas na ułożenie w pozycji bezpiecznej</Polish>
+        </Key>
+        <Key ID="STR_kat_airway_TIME_RECOVERY_DESC">
+            <English>How long it will take to establish recovery position</English>
```

Comment:
```suggestion
            <English>How long it will take to establish recovery position</English>
            <German>Die Zeit, wie lange es dauert eine erfolgreiche stabile Seitenlage durchzuführen</German>
```